immudb/datatypes.py

```diff
@@ -11,7 +11,13 @@
 # limitations under the License.
 
 from dataclasses import dataclass
-import struct
+from enum import IntEnum
+
+
+class TxMode(IntEnum):
```

Comment:
from immudb.grpc.schema_pb2 import ReadOnly, WriteOnly, ReadWrite

Reply:
Done in https://github.com/codenotary/immudb-py/pull/46/commits/ecd9c5b216fb19f0e3c761c5b7ad2f7788c01342